specs-go/v2/descriptor.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 The Linux Foundation

```

Comment:
I wonder, shouldn't the descriptor be vendor from image spec..

Reply:
The motivation for moving the descriptor here is to keep the artifact spec independent of the container runtime (image-spec). 
Here's an image illustrating the idea of making an artifact a "top level" manifest: https://github.com/notaryproject/artifacts/blob/prototype-2/media/artifacts-spec-objects.png. @SteveLasker , is there a document that expounds on this idea?